WalletWasabi.Fluent/Helpers/TransactionHelpers.cs

```diff
@@ -90,6 +94,51 @@ public static BuildTransactionResult BuildTransaction(Wallet wallet, Transaction
 			tryToSign: tryToSign);
 	}
 
+	public static bool TryBuildTransaction(
+		KeyManager keyManager,
+		TransactionInfo transactionInfo,
+		ICoinsView allCoins,
+		IEnumerable<SmartCoin> allowedCoins,
+		string password,
+		out Money minimumAmount)
+	{
+		minimumAmount = Money.Zero;
+
+		try
+		{
+			var intent = new PaymentIntent(
+				destination: transactionInfo.Destination,
+				amount: transactionInfo.Amount,
+				subtractFee: transactionInfo.SubtractFee,
+				label: transactionInfo.UserLabels);
+
+			var network = keyManager.GetNetwork();
+			string dataDir = EnvironmentHelpers.GetDataDir(Path.Combine("WalletWasabi", "DummyDirectory"));
+			var emptyTransactionStore = new AllTransactionStore(dataDir, network);
+			var builder = new TransactionFactory(network, keyManager, allCoins, emptyTransactionStore, password, true);
```

Comment:
I got these in the logs: 

```
2022-10-12 15:07:28.041 [1] INFO        EnvironmentHelpers.GetDataDir (71)      Creating data directory at `C:\Users\user\AppData\Roaming\WalletWasabi\DummyDirectory`.
2022-10-12 15:07:28.167 [1] INFO        NBitcoinExtensions.AddPrevTxs (444)     Transaction id: 8aaec8d0acfedd994f3ddfa20f514b2d45096da81f3da67c03f8648763b865ac is missing from the transactionStore. Ignoring...
```

You need to create that interface, we were talking about. It is not worth touching AllTransactionStore.

Reply:
@molnard I pushed changes, but the this log message still comes:
```
2022-10-13 10:01:54.413 [1] INFO        NBitcoinExtensions.AddPrevTxs (444)     Transaction id: cededb1c2fc2e872e07cbdb602b3b6b33f27f8856ebf1b71d63d4aa8f9b667f4 is missing from the transactionStore. Ignoring...
```

Due to this log: https://github.com/zkSNACKs/WalletWasabi/blob/master/WalletWasabi/Extensions/NBitcoinExtensions.cs#L444

Should I change it to `Debug` level?

Reply:
https://github.com/zkSNACKs/WalletWasabi/pull/9289/commits/27e9d13026b266dd756a272f0a4481b6d219e88a